runtime/connectors/file/file.go

```diff
@@ -67,3 +69,7 @@ type connector struct{}
 func (c connector) Spec() connectors.Spec {
 	return spec
 }
+
+func (c connector) ConsumeAsFile(ctx context.Context, source *connectors.Source, callback func(filename string) error) error {
+	return errdefs.NotImplemented(nil)
```

Comment:
This adds a dependency on `"github.com/docker/docker/errdefs"`. Better to do `errors.New("not implemented")`.